custom/icds/tests/models/test_ccz_hosting.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import mock
+from django.test import TestCase
+from django.core.exceptions import ValidationError
+
+from custom.icds.models import (
+    CCZHostingLink,
+    CCZHosting,
+)
+
+
+@mock.patch('custom.icds.tasks.setup_ccz_file_for_hosting.delay')
```

Comment:
FWIW you can pass `MagicMock()` here as an arg so it doesn't pass the default mock object to your methods

Reply:
Oh, I see you did that for some tests later

Reply:
i think i simply ignored the mocks i didn't care for by `*_`. An object would be created anyway right?

Reply:
Yeah, it's the same except for whether the mocks get passed to the function.